templates/includes/records/archive-description.html

```diff
@@ -59,7 +59,7 @@ <h2>Summary of collections</h2>
                     <ul>
                         {% for collection in record.archive_collections.collection_info_list %}
                             {% if collection.count > 0 %}
-                                <li><a href="">{{ collection.display_name }}({{ collection.count }})</a></li>
+                                <li><a href="#heading-{{collection.name}}">{{ collection.display_name }} ({{ collection.count }})</a></li>
```

Comment:
Sorry @TNA-Allan, one last thing... is there a reason why we use `display_name` here, but `long_display_name` above?

Reply:
@andy 
One of the [collections ](https://github.com/nationalarchives/ds-wagtail/blob/3a374dcd2fa1033718ccc2c41e0f0f0b5b024dc7/etna/ciim/constants.py#L714-L755)(NRA records) have a `long_display_name` to show in the Collections Information (below) area and `display_name` (above) in the summary area

Reply:
For other collections, both values `long_display_name, display_name` are same.